configuration/backend_configuration/ampathforms/surgery-core_demo.json

```diff
@@ -170,7 +170,7 @@
               "label": "Time of Procedure",
               "type": "obs",
               "questionOptions": {
-                "rendering": "date",
+                "rendering": "datetime",
                 "concept": "0934658e-fee3-44e3-a8f2-292dc3587f54",
                 "weeksList": ""
               },
```

Comment:
```suggestion
              },
              "datePickerFormat": "both",
```

Reply:
Since the form engine uses this parameter under the hood to figure out whether to render either the calendar, or timer or both. Just the change you've done suffices, but `datePickerFormat` is preferred to set the date picker preference.